xla/stream_executor/gpu/gpu_driver.h

```diff
@@ -346,6 +346,49 @@ class GpuDriver {
                                        GpuGraphHandle graph,
                                        const GraphInstantiateFlags& flags);
 
+  // Graph instantiation parameters
+  struct GraphInstantiateParams {
+    GraphInstantiateFlags flags;
+    GpuGraphNodeHandle err_node_out = NULL;
+    GpuStreamHandle upload_stream = NULL;
+#if CUDA_VERSION >= 12000
```

Comment:
Could you please add `#include "third_party/gpus/cuda/include/cuda.h"` for `CUDA_VERSION`?

Reply:
Fixed, could you help make the merge?

Reply:
@shawnwang18 Eugene is right, gpu_driver.h should be platform agnostic, so no Nvidia specific #ifdef is allowed here.